apps/web/src/common/key-map.ts

```diff
@@ -165,6 +166,15 @@ const KEYMAP = [
   //     themestore.get().toggleNightMode();
   //   },
   // },
+  {
+    keys: ["ctrl+k"],
```

Comment:
Add `cmd+k` as well. Also `ctrl+shift+k` should open the command palette in search mode.

Reply:
this PR has `ctrl+shift+k` for inserting links, do you have a suggestion for that?

Reply:
Maybe we can do `ctrl+p` for search mode but that won't work on browsers, I think?

Reply:
> won't work on browsers,

That's what I thought, but it works for some reason 😆 

Command palette trigger shortcuts now:
`ctrl+k` - open in command mode
`ctrl+p` - open in search mode

I think these are good for now, though I'll keep looking for better. If you have any suggestions, do let me know